custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
+
+async def _compile_static_switches(
+    config_entry: ConfigEntry,
+    coordinator: OPNsenseDataUpdateCoordinator,
+    state: Mapping[str, Any],
+) -> list:
+    entities: list = []
+    entity = OPNsenseUnboundBlocklistSwitch(
+        config_entry=config_entry,
+        coordinator=coordinator,
+        entity_description=SwitchEntityDescription(
+            key=f"unbound_blocklist.switch",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)